completion/hyperref.cwl

```diff
@@ -129,8 +129,8 @@ xetex
 \hyperref[label%ref]{URL}{category}{name}{text}#M
 \hyperref[label%ref]{text}
 \hyperref{text}
-\hyperlink{name}{text}
-\hypertarget{name}{text}
+\hyperlink{target name}{link text}
+\hypertarget{target name}{anchor text}
```

Comment:
i would assume that spell checking is now disabled for link/anchor text ... %text?

Reply:
Ah yes. Updated. I often forget those reserved argument names.